models/articles-model.js

```diff
@@ -18,6 +18,18 @@ function retrieveArticles(article_id) {
     })
 }
 
+function removeComment(comment_id) {
+    return connection.query(`DELETE FROM comments
+                            WHERE comment_id = $1
+                            RETURNING *;`, [comment_id])
+        .then(({ rows }) => {
+            if (!rows.length) {
+                return Promise.reject({status: 404, message: "comment does not exist"})
+            } else {
+                return Promise.resolve()
+            }
```

Comment:
I think a `return` on it's own would also do the job here but I actually quite like the explicit way you've written this using _Promise.resolve_, very readable